#Request data
---
#Response data
string[] obj_list
string[] tagID
geometry_msgs/Pose[] pose